### PR TITLE
Fix CodeAction

### PR DESCRIPTION
**Pull request by karlhorky**
Fix CodeAction introduced in #68

New config for `Settings (json)`:

```json5
  // Sort package.json keys
  "editor.codeActionsOnSave": ["source.sortPackageJson"],
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,9 @@
       }
     ]
   },
+  "activationEvents": [
+    "onStartupFinished"
```
Comment (karlhorky) [inline package.json:45]: This is needed so that the extension can format right away, before the first run of the command from the Command Palette

Comment (unional) [reply package.json:45]: Thanks, that make sense.

**Comment by unional** [comment]
@karlhorky thanks for your contribution! 🎉

**Comment by karlhorky** [comment]
Thanks for the review and merge @unional ! I guess this will be released soon as `1.4.3`? I will test it and then add a short section in the documentation so that others know how to set up the sorting on save.

**Comment by unional** [comment]
🤣 completely forgot about that. Get so used to using `semantic-release` and/or `changesets`.

**Comment by karlhorky** [comment]
Great, `unional.vscode-sort-package-json@1.4.3` works! Opened a PR for documentation:

- https://github.com/unional/vscode-sort-package-json/pull/75

**Pull request merged**